.github/workflows/create-release-pr.yml

```
@@ -0,0 +1,81 @@
+name: Manual Create Release PR
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Release type'
+        required: true
+        default: 'patch'
+        type: choice
+        options:
+          - minor
+          - patch
+
+jobs:
+  create-release-pr:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Fail if branch is not develop
+        if: github.event_name == 'workflow_dispatch' && github.ref != 'refs/heads/develop'
+        run: |
+          echo "This workflow should not be triggered on a branch other than develop"
+          exit 1
+
+      - name: Check out code
+        uses: actions/checkout@v2
+
+      - name: Set Tegel user
+        run: git config --global user.name "Tegel - Scania" && git config --global user.email "tegel.design.system@gmail.com"
+
+      - name: Core - Bump version
+        run: npm version ${{ github.event.inputs.version }}
+        working-directory: packages/core
+        id: core-version
```

```suggestion
      - name: Core - Bump version
        id: core-version
        working-directory: packages/core
        run: npm version ${{ github.event.inputs.version }}
```

What do you think if we flip order of commands. Machine won't care, but for human it might be easier first to understand id, working-directory and then command to be executed?

If you agree, please apply change to all scripts.

---

I think what's most readable depends what you're looking for in this case. I'll switch!